platforms/cpuinfo_linux_arm_test.go

```diff
@@ -0,0 +1,33 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package platforms
+
+func TestGetCPUArchitecture(t *testing.T) {
+	p, err := getCPUArchitecture()
+	if err != nil {
+		t.Fatal(err)
+	}
+	variants := []string{"v8", "v7", "v6", "v5", "v4", "v3"}
+	for _, variant := range variants {
+		if p == variant {
+			t.Logf("got valid variant as expected: %#v = %#v\n", p, variant)
```

Comment:
On what CPU did you run this test?

Reply:
in arm cpu run this test.

Reply:
On what arm? Could you provide uname -a and /proc/cpuinfo ?

Reply:
remove this test. use `TestCPUVariant_MonkeyNoCpuArchitecture` to test `getCPUArchitecture` func.